docs/supported-libraries.md

```diff
@@ -19,119 +19,123 @@ These are the supported libraries and frameworks:
-| [Vert.x RxJava2](https://vertx.io/docs/vertx-rx/java2/)                                                                                     | 3.5+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     |                                                                                        |
-| [Vibur DBCP](https://www.vibur.org/)                                                                                                        | 11.0+                         | [opentelemetry-vibur-dbcp-11.0](../instrumentation/vibur-dbcp-11.0/library)                                                                                                                                                                                                                                                                                                             |                                                                                        |
+| [Undertow](https://undertow.io/)                                                                                                            | 1.4+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [HTTP Server Spans], [HTTP Server Metrics]                                             |
+| [Vaadin](https://vaadin.com/)                                                                                                               | 14.2+                         | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [Source Code Attributes]                                                               |
```

Comment:
```suggestion
| [Vaadin](https://vaadin.com/)                                                                                                               | 14.2+                         | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Provides `http.route` [2], Controller Spans [3]                                                                |
```

Reply:
Good catch. It should probably be "Provides `http.route` [2], Controller Spans [3], [Source Code Attributes]", right? Or are you suggesting that `[Source Code Attributes]` isn't worth including?

Reply:
I'm not sure `[Source Code Attributes]` is worth including, I think the important thing here is what spans are emitted.

I think `http.route` is a special case b/c of its importance to good span names and also its importance for metrics.